### PR TITLE
[OCPQE-26407] MCO prow jobs for older version

### PR DESCRIPTION
**Pull request by ptalgulk01**
[OCPQE-26407](https://issues.redhat.com/browse/OCPQE-26407)
- Remove the MCO  from part2-3 job 
    -  As it explodes the cluster in 4.18-4.16 and this version already has dedicated jobs
    - Implemented MCO Dedicated MCO job for 4.13-4.15
- Created proxy job for 4.13-4.15

### CODE REVIEW

**Comment by ptalgulk01** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-longduration-mco-fips-proxy-all-f14 periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-longduration-mco-fips-proxy-all-f14 periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-longduration-mco-fips-proxy-all-f14

**Comment by ptalgulk01** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-arm64-nightly-aws-ipi-f28-longduration-mco-all periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-nightly-aws-ipi-f28-longduration-mco-all periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-f28-longduration-mco-all

**Comment by ptalgulk01** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-f28-longduration-mco-all

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 1]
```diff
@@ -281,7 +281,22 @@ tests:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      TEST_SCENARIOS: MCO
     test:
     - chain: openshift-e2e-test-qe-longrun
```
Comment (sergiordlr) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:301]: We shouldn't use `openshift-e2e-test-qe-longrun`. We should use this chain instead:
```
    test:
    - chain: openshift-e2e-test-mco-qe-longrun
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml [hunk 2]
```diff
@@ -293,7 +293,22 @@ tests:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      TEST_SCENARIOS: MCO
     test:
     - chain: openshift-e2e-test-qe-longrun
```
Comment (sergiordlr) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml:313]: We shouldn't use `openshift-e2e-test-qe-longrun`. We should use this chain instead:
```
    test:
    - chain: openshift-e2e-test-mco-qe-longrun
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml [hunk 3]
```diff
@@ -293,7 +293,22 @@ tests:
...
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      TEST_SCENARIOS: MCO
     test:
     - chain: openshift-e2e-test-qe-longrun
```
Comment (sergiordlr) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:313]: We shouldn't use `openshift-e2e-test-qe-longrun`. We should use this chain instead:
```
    test:
    - chain: openshift-e2e-test-mco-qe-longrun
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml (hunk 1 above)
Comment (ptalgulk01) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml]: done, thankyou!

**Comment by sergiordlr** [comment]
Hello @aleskandro ! We are adding some mco dedicated jobs to adm and arm. Could you review the configurations that we have added, please?

Thanks!

**Comment by sergiordlr** [comment]
Hello @lwan-wanglin ! This PR is adding some new jobs to ARM,  could you tell us if it looks good to you?
Thank you!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 4]
```diff
@@ -281,10 +281,25 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       OCP_ARCH: arm64
-      TEST_SCENARIOS: MCO|Cluster_Infrastructure
+      TEST_SCENARIOS: Cluster_Infrastructure
     test:
     - chain: openshift-e2e-test-qe-longrun
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-f28-longduration-mco-all
+  cron: 11 2 4,24 * *
```
Comment (lwan-wanglin) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:289]: the cron frequency is inconsistent with the f28 in the job name, see here https://github.com/openshift/release/tree/master/ci-operator/config/openshift/openshift-tests-private#job-frequency
and also below new added arm64 jobs, others look good to me.

**Comment by lwan-wanglin** [comment]
cc @liangxia to review, thanks!

**Pull request merged**